### PR TITLE
Split x11 regression test and console regression in the regression group

### PR DESCRIPTION
**Pull request by tinawang123**
Split x11 regression test and console regression in the regression
group. To avoid some performance issue.

- Related ticket: https://progress.opensuse.org/issues/104688
- Verification run: 
https://openqa.suse.de/tests/7964620 (x86_64 console)
https://openqa.suse.de/tests/7968222 (x86_64 x11)
https://openqa.suse.de/tests/7971896 (s390 x11)
https://openqa.suse.de/tests/7967923 (s390 console)
https://openqa.suse.de/tests/7967917 (aarch64 console)
https://openqa.suse.de/tests/7971823 (aarch64 x11)
https://openqa.suse.de/tests/7968028 (ppc64le x11)
https://openqa.suse.de/tests/7967920 (ppc64le console)

### CODE REVIEW

**Comment by lemon-suse** [comment]
'#means regression test part2 include x11 test and some performance related test.'
I think need update the description since now only x11 test in part2.

File: schedule/migration/x86_regression_test_online.yaml [hunk 1]
```diff
@@ -79,17 +81,12 @@ conditional_schedule:
         - console/sshd
         - console/ssh_cleanup
         - console/mtab
+        - console/upgrade_snapshots
         - console/zypper_lifecycle
         - console/orphaned_packages_check
-        - console/kdump_and_crash
```
Comment (lemon-suse) [inline schedule/migration/x86_regression_test_online.yaml:84]: Please don't remove it, this module we added for kdump.

Comment (tinawang123) [reply schedule/migration/x86_regression_test_online.yaml:84]: Added

**Review by lemon-suse — APPROVED** [review]

**Review by chcao — APPROVED** [review]
LGTM

**Comment by coolgw** [comment]
@tinawang123  seems conflict happen need resolve

**Comment by tinawang123** [comment]
> @tinawang123 seems conflict happen need resolve

resolved now

**Pull request merged**